test/cpp/interop/xds_interop_client.cc

```diff
@@ -67,6 +67,8 @@ ABSL_FLAG(int32_t, stats_port, 50052,
           "Port to expose peer distribution stats service.");
 ABSL_FLAG(std::string, rpc, "UnaryCall",
           "a comma separated list of rpc methods.");
+ABSL_FLAG(int32_t, request_payload_size, 0, "request payload size in bytes.");
+ABSL_FLAG(int32_t, response_payload_size, 0, "response payload size in bytes.");
```

Comment:
Ok, apparently this field results in an error right now: https://github.com/grpc/grpc/blob/e36025d3d24182d0c512b767a8e3687ae2fd9640/test/cpp/interop/interop_server.cc#L213C14-L216

Let's avoid it until it's implemented in the server.

Reply:
Ooops, misread it. Deleting this thread.